harvester/database/models.py

```diff
@@ -0,0 +1,58 @@
+from sqlalchemy import text, String, Integer,  ARRAY, ForeignKey, DateTime
+from sqlalchemy.orm import DeclarativeBase, relationship, mapped_column
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class Base(DeclarativeBase):
+    id = mapped_column(
+        UUID(as_uuid=True), primary_key=True, server_default=text("gen_random_uuid()")
+    )
+
+class HarvestSource(Base):
+    __tablename__ = 'harvest_source'
+    
+    name = mapped_column(String, nullable=False)
+    notification_emails = mapped_column(ARRAY(String))
+    organization_name = mapped_column(String)
```

Comment:
change to `organization_id = mapped_column(String, nullable=False)`?

Reply:
done.

Reply:
agree 100%. with the addition of the `organization` table we should also add a FK.